tests/notebook/test_demo_notebook.ipynb

```diff
@@ -1792,6 +1800,7 @@
     }
    ],
    "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
```

Comment:
Should we also mention Pandas 1.5 here, as done below?

Reply:
```suggestion
    "# NBVAL_IGNORE_OUTPUT\n",
    "# The ignore statement above is because of output difference between Pandas 1 and 2\n",
    "# and can be removed once Pandas 1 support is dropped\n",
```